keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/ApplicationLoadBalancerHandler.kt

```diff
@@ -58,6 +59,8 @@ class ApplicationLoadBalancerHandler(
     val description = "$action ${resource.kind} load balancer ${resource.spec.moniker.name} in " +
       "${resource.spec.location.accountName}/${resource.spec.location.region}"
 
+    val notifications = environmentResolver.getNotificationsFor(resource.id)
```

Comment:
I thought it was nicer to put things that dealt with fetching environments and re-shaping the contents in one place where all the handlers could use it.